src/kiota/KiotaPluginCommands.cs

```diff
@@ -70,6 +70,15 @@ internal static Option<bool> GetNoWorkspaceOption()
         noWorkspaceOption.AddAlias("--nw");
         return noWorkspaceOption;
     }
+    internal static Option<List<string>> GetOverlaysOption()
```

Comment:
I don't think we need to duplicate the method here, we should be able to get it from the host